src/components/board/board.test.js

```diff
@@ -1,8 +1,36 @@
+const board = require('../../services/boardService');
 
+const FILES = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'];
+const RANKS = [1, 2, 3, 4, 5, 6, 7, 8];
 
+// testing to make sure the array is populated with 64 object values
```

Comment:
Just my nit-pickiness - can you wrap these tests in a describe block? 

describe('board.js', () => {
....
});

Reply:
Sure thing